app/containers/AddProfile/index.js

```diff
@@ -84,13 +87,35 @@ const AddProfile = () => {
     return true;
   };
 
+  const handleFetchRfProfiles = e => {
+    if (rfProfiles.getAllProfiles.context.lastPage) {
+      return false;
+    }
+
+    e.persist();
+    const { target } = e;
+
+    if (target.scrollTop + target.offsetHeight === target.scrollHeight) {
+      fetchMoreRfProfiles({
+        variables: { context: { ...rfProfiles.getAllProfiles.context } },
+        updateQuery: updateQueryGetAllProfiles,
+      });
+    }
+
+    return true;
+  };
+
   return (
     <AddProfilePage
       onCreateProfile={handleAddProfile}
       ssidProfiles={
         (ssidProfiles && ssidProfiles.getAllProfiles && ssidProfiles.getAllProfiles.items) || []
       }
+      rfProfiles={
+        (rfProfiles && rfProfiles.getAllProfiles && rfProfiles.getAllProfiles.items) || []
```

Comment:
switch to `rfProfiles={rfProfiles?.getAllProfiles?.items}`
same for the prop `ssidProfiles`